NOTICES.md

```diff
@@ -0,0 +1,42 @@
+# Notices for OpenJCEPlus Module
```

Comment:
Please remove this NOTICES.md file.

Reply:
It's just for building a personal jdk. will remove it after the test is done.